core/modules/views/plugins/views_plugin_display.inc

```diff
@@ -1970,13 +1971,13 @@ class views_plugin_display extends views_plugin {
         }
         else {
           $output .= '<p>' . t('This is the default theme template used for this display.') . '</p>';
-          $output .= '<pre>' . check_plain(file_get_contents('./' . $this->definition['theme path'] . '/' . strtr($this->definition['theme'], '_', '-') . '.tpl.php')) . '</pre>';
+          $output .= '<textarea rows="15" cols="82" class="code-sample">' . check_plain(file_get_contents('./' . $this->definition['theme path'] . '/' . strtr($this->definition['theme'], '_', '-') . '.tpl.php')) . '</textarea>';
```

Comment:
The textareas need `readonly="readonly"` attribute on each one to minimize security risk.

Reply:
good idea!